portal/templates/_macros/status-label.njk

```diff
@@ -0,0 +1,19 @@
+{% set statusLabelMap = {
+    DRAFT: "Draft",
+    BANK_CHECK: "Ready for Checker's approval",
+    CHANGES_REQUIRED: "Further Maker's input required",
+    ABANDONED: "Abandoned Deal",
+    SUBMITTED_TO_UKEF: "Submitted",
+    UKEF_ACKNOWLEDGED: "Acknowledged by UKEF",
+    UKEF_IN_PROGRESS: "In progress by UKEF",
+    UKEF_ACCEPTED_CONDITIONAL: "Accepted by UKEF (with conditions)",
+    UKEF_ACCEPTED_UNCONDITIONAL: "Accepted by UKEF (without conditions)",
+    UKEF_DECLINED: "Rejected by UKEF",
+    EXPIRED: "Expired",
+    WITHDRAWN: "Withdrawn"
+  }
+%}
+
+{% macro getLabel(status) %}
```

Comment:
Would this not be better as a filter than a macro? much easier to use via pipe function I think

Reply:
I can change it if you feel strongly enough ... just say the word

Reply:
Given the reuse elsewhere definitely feels the best way - sorry to be annoying about it

Reply:
its cool .... will make it so